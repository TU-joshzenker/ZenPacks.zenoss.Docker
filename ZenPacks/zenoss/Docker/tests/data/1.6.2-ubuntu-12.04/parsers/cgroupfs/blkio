
/sys/fs/cgroup/blkio/

/sys/fs/cgroup/blkio/docker

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_queued_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_merged_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_wait_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_service_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_serviced_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_service_bytes_recursive
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.sectors_recursive

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.time_recursive

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_queued
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_merged
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_wait_time
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_service_time
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_serviced
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.io_service_bytes
Total 0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.sectors

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.time

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.leaf_weight
500

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.leaf_weight_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.weight
500

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.weight_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.io_serviced
202:1 Read 329
202:1 Write 0
202:1 Sync 0
202:1 Async 329
202:1 Total 329
Total 329

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.io_service_bytes
202:1 Read 6451200
202:1 Write 0
202:1 Sync 0
202:1 Async 6451200
202:1 Total 6451200
Total 6451200

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.write_iops_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.read_iops_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.write_bps_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.throttle.read_bps_device

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/blkio.reset_stats

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/notify_on_release
0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/tasks
1648
1667

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.clone_children
0

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.event_control

/sys/fs/cgroup/blkio/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.procs
1648
1667

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_queued_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_merged_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_wait_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_service_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_serviced_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_service_bytes_recursive
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.sectors_recursive

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.time_recursive

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_queued
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_merged
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_wait_time
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_service_time
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_serviced
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.io_service_bytes
Total 0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.sectors

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.time

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.leaf_weight
500

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.leaf_weight_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.weight
500

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.weight_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.io_serviced
202:1 Read 103
202:1 Write 1918
202:1 Sync 1918
202:1 Async 103
202:1 Total 2021
Total 2021

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.io_service_bytes
202:1 Read 2240512
202:1 Write 7856128
202:1 Sync 7856128
202:1 Async 2240512
202:1 Total 10096640
Total 10096640

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.write_iops_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.read_iops_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.write_bps_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.throttle.read_bps_device

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/blkio.reset_stats

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/notify_on_release
0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/tasks
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.clone_children
0

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.event_control

/sys/fs/cgroup/blkio/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.procs
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/blkio/docker/blkio.io_queued_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_merged_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_wait_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_service_time_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_serviced_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_service_bytes_recursive
Total 0

/sys/fs/cgroup/blkio/docker/blkio.sectors_recursive

/sys/fs/cgroup/blkio/docker/blkio.time_recursive

/sys/fs/cgroup/blkio/docker/blkio.io_queued
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_merged
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_wait_time
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_service_time
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_serviced
Total 0

/sys/fs/cgroup/blkio/docker/blkio.io_service_bytes
Total 0

/sys/fs/cgroup/blkio/docker/blkio.sectors

/sys/fs/cgroup/blkio/docker/blkio.time

/sys/fs/cgroup/blkio/docker/blkio.leaf_weight
500

/sys/fs/cgroup/blkio/docker/blkio.leaf_weight_device

/sys/fs/cgroup/blkio/docker/blkio.weight
500

/sys/fs/cgroup/blkio/docker/blkio.weight_device

/sys/fs/cgroup/blkio/docker/blkio.throttle.io_serviced
202:1 Read 0
202:1 Write 0
202:1 Sync 0
202:1 Async 0
202:1 Total 0
Total 0

/sys/fs/cgroup/blkio/docker/blkio.throttle.io_service_bytes
202:1 Read 0
202:1 Write 0
202:1 Sync 0
202:1 Async 0
202:1 Total 0
Total 0

/sys/fs/cgroup/blkio/docker/blkio.throttle.write_iops_device

/sys/fs/cgroup/blkio/docker/blkio.throttle.read_iops_device

/sys/fs/cgroup/blkio/docker/blkio.throttle.write_bps_device

/sys/fs/cgroup/blkio/docker/blkio.throttle.read_bps_device

/sys/fs/cgroup/blkio/docker/blkio.reset_stats

/sys/fs/cgroup/blkio/docker/notify_on_release
0

/sys/fs/cgroup/blkio/docker/tasks

/sys/fs/cgroup/blkio/docker/cgroup.clone_children
0

/sys/fs/cgroup/blkio/docker/cgroup.event_control

/sys/fs/cgroup/blkio/docker/cgroup.procs

/sys/fs/cgroup/blkio/blkio.io_queued_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.io_merged_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.io_wait_time_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.io_service_time_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.io_serviced_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.io_service_bytes_recursive
Total 0

/sys/fs/cgroup/blkio/blkio.sectors_recursive

/sys/fs/cgroup/blkio/blkio.time_recursive

/sys/fs/cgroup/blkio/blkio.io_queued
Total 0

/sys/fs/cgroup/blkio/blkio.io_merged
Total 0

/sys/fs/cgroup/blkio/blkio.io_wait_time
Total 0

/sys/fs/cgroup/blkio/blkio.io_service_time
Total 0

/sys/fs/cgroup/blkio/blkio.io_serviced
Total 0

/sys/fs/cgroup/blkio/blkio.io_service_bytes
Total 0

/sys/fs/cgroup/blkio/blkio.sectors

/sys/fs/cgroup/blkio/blkio.time

/sys/fs/cgroup/blkio/blkio.leaf_weight
1000

/sys/fs/cgroup/blkio/blkio.leaf_weight_device

/sys/fs/cgroup/blkio/blkio.weight
1000

/sys/fs/cgroup/blkio/blkio.weight_device

/sys/fs/cgroup/blkio/blkio.throttle.io_serviced
202:16 Read 389
202:16 Write 1
202:16 Sync 1
202:16 Async 389
202:16 Total 390
202:1 Read 6381
202:1 Write 10746
202:1 Sync 9084
202:1 Async 8043
202:1 Total 17127
7:7 Read 0
7:7 Write 0
7:7 Sync 0
7:7 Async 0
7:7 Total 0
7:6 Read 0
7:6 Write 0
7:6 Sync 0
7:6 Async 0
7:6 Total 0
7:5 Read 0
7:5 Write 0
7:5 Sync 0
7:5 Async 0
7:5 Total 0
7:4 Read 0
7:4 Write 0
7:4 Sync 0
7:4 Async 0
7:4 Total 0
7:3 Read 0
7:3 Write 0
7:3 Sync 0
7:3 Async 0
7:3 Total 0
7:2 Read 0
7:2 Write 0
7:2 Sync 0
7:2 Async 0
7:2 Total 0
7:1 Read 0
7:1 Write 0
7:1 Sync 0
7:1 Async 0
7:1 Total 0
7:0 Read 0
7:0 Write 0
7:0 Sync 0
7:0 Async 0
7:0 Total 0
1:15 Read 166
1:15 Write 0
1:15 Sync 0
1:15 Async 166
1:15 Total 166
1:14 Read 166
1:14 Write 0
1:14 Sync 0
1:14 Async 166
1:14 Total 166
1:13 Read 166
1:13 Write 0
1:13 Sync 0
1:13 Async 166
1:13 Total 166
1:12 Read 166
1:12 Write 0
1:12 Sync 0
1:12 Async 166
1:12 Total 166
1:11 Read 166
1:11 Write 0
1:11 Sync 0
1:11 Async 166
1:11 Total 166
1:10 Read 166
1:10 Write 0
1:10 Sync 0
1:10 Async 166
1:10 Total 166
1:9 Read 166
1:9 Write 0
1:9 Sync 0
1:9 Async 166
1:9 Total 166
1:8 Read 166
1:8 Write 0
1:8 Sync 0
1:8 Async 166
1:8 Total 166
1:7 Read 166
1:7 Write 0
1:7 Sync 0
1:7 Async 166
1:7 Total 166
1:6 Read 166
1:6 Write 0
1:6 Sync 0
1:6 Async 166
1:6 Total 166
1:5 Read 166
1:5 Write 0
1:5 Sync 0
1:5 Async 166
1:5 Total 166
1:4 Read 166
1:4 Write 0
1:4 Sync 0
1:4 Async 166
1:4 Total 166
1:3 Read 166
1:3 Write 0
1:3 Sync 0
1:3 Async 166
1:3 Total 166
1:2 Read 166
1:2 Write 0
1:2 Sync 0
1:2 Async 166
1:2 Total 166
1:1 Read 166
1:1 Write 0
1:1 Sync 0
1:1 Async 166
1:1 Total 166
1:0 Read 166
1:0 Write 0
1:0 Sync 0
1:0 Async 166
1:0 Total 166
Total 20173

/sys/fs/cgroup/blkio/blkio.throttle.io_service_bytes
202:16 Read 1590272
202:16 Write 4096
202:16 Sync 4096
202:16 Async 1590272
202:16 Total 1594368
202:1 Read 85390336
202:1 Write 44437504
202:1 Sync 37294080
202:1 Async 92533760
202:1 Total 129827840
7:7 Read 0
7:7 Write 0
7:7 Sync 0
7:7 Async 0
7:7 Total 0
7:6 Read 0
7:6 Write 0
7:6 Sync 0
7:6 Async 0
7:6 Total 0
7:5 Read 0
7:5 Write 0
7:5 Sync 0
7:5 Async 0
7:5 Total 0
7:4 Read 0
7:4 Write 0
7:4 Sync 0
7:4 Async 0
7:4 Total 0
7:3 Read 0
7:3 Write 0
7:3 Sync 0
7:3 Async 0
7:3 Total 0
7:2 Read 0
7:2 Write 0
7:2 Sync 0
7:2 Async 0
7:2 Total 0
7:1 Read 0
7:1 Write 0
7:1 Sync 0
7:1 Async 0
7:1 Total 0
7:0 Read 0
7:0 Write 0
7:0 Sync 0
7:0 Async 0
7:0 Total 0
1:15 Read 679936
1:15 Write 0
1:15 Sync 0
1:15 Async 679936
1:15 Total 679936
1:14 Read 679936
1:14 Write 0
1:14 Sync 0
1:14 Async 679936
1:14 Total 679936
1:13 Read 679936
1:13 Write 0
1:13 Sync 0
1:13 Async 679936
1:13 Total 679936
1:12 Read 679936
1:12 Write 0
1:12 Sync 0
1:12 Async 679936
1:12 Total 679936
1:11 Read 679936
1:11 Write 0
1:11 Sync 0
1:11 Async 679936
1:11 Total 679936
1:10 Read 679936
1:10 Write 0
1:10 Sync 0
1:10 Async 679936
1:10 Total 679936
1:9 Read 679936
1:9 Write 0
1:9 Sync 0
1:9 Async 679936
1:9 Total 679936
1:8 Read 679936
1:8 Write 0
1:8 Sync 0
1:8 Async 679936
1:8 Total 679936
1:7 Read 679936
1:7 Write 0
1:7 Sync 0
1:7 Async 679936
1:7 Total 679936
1:6 Read 679936
1:6 Write 0
1:6 Sync 0
1:6 Async 679936
1:6 Total 679936
1:5 Read 679936
1:5 Write 0
1:5 Sync 0
1:5 Async 679936
1:5 Total 679936
1:4 Read 679936
1:4 Write 0
1:4 Sync 0
1:4 Async 679936
1:4 Total 679936
1:3 Read 679936
1:3 Write 0
1:3 Sync 0
1:3 Async 679936
1:3 Total 679936
1:2 Read 679936
1:2 Write 0
1:2 Sync 0
1:2 Async 679936
1:2 Total 679936
1:1 Read 679936
1:1 Write 0
1:1 Sync 0
1:1 Async 679936
1:1 Total 679936
1:0 Read 679936
1:0 Write 0
1:0 Sync 0
1:0 Async 679936
1:0 Total 679936
Total 142301184

/sys/fs/cgroup/blkio/blkio.throttle.write_iops_device

/sys/fs/cgroup/blkio/blkio.throttle.read_iops_device

/sys/fs/cgroup/blkio/blkio.throttle.write_bps_device

/sys/fs/cgroup/blkio/blkio.throttle.read_bps_device

/sys/fs/cgroup/blkio/blkio.reset_stats

/sys/fs/cgroup/blkio/release_agent


/sys/fs/cgroup/blkio/notify_on_release
0

/sys/fs/cgroup/blkio/tasks
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
874
875
876
901
908
918
919
922
933
934
942
969
972
973
990
1090
1091
1195
1199
1200
1201
1202
1204
1206
1260
1360
1361
1465
1469
1470
1544
1557
1561
1574
1575
1584
1625
1629
1636
1637
1638
1639
1643
1644
1645
1646
1647
1653
1666
2414
3113
3620
3917
3950
4130
4134
4264

/sys/fs/cgroup/blkio/cgroup.sane_behavior
0

/sys/fs/cgroup/blkio/cgroup.clone_children
0

/sys/fs/cgroup/blkio/cgroup.event_control

/sys/fs/cgroup/blkio/cgroup.procs
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
901
908
918
919
922
933
934
942
969
972
990
1090
1091
1195
1199
1260
1360
1361
1465
1469
1470
1575
1584
1636
1643
2414
3113
3620
3950
4130
4134
4268
