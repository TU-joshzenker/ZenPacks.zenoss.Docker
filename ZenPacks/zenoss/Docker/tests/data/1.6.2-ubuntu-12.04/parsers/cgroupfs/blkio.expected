# vim: ft=yaml
command: '/usr/bin/env sudo find /sys/fs/cgroup/blkio/ -printf \"\n%p\n" -exec cat {} 2>/dev/null \;'

values:
  9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f:
    bytes_total: 10096640
    bytes_read: 2240512
    bytes_write: 7856128
    io_total: 2021
    io_read: 103
    io_write: 1918
